### DISCUSSION TITLE
Micro Frontend - Single SPA

### DISCUSSION THREAD

**Discussion by KamranKhan-Dev** [Budibase/budibase#4039] [Ideas]
Would you consider the ability to produce an output that can be integrated with the container of a micro frontend web application?

I'm currently working on a portal that have contributors from different teams working at their own different paces with limited hands-on frontend coding experience.  The central team is experienced, but cannot accommodate demand from all the different product teams.  With the central team in charge of the micro frontend container, it would empower the product teams to use Budibase as a low-code environment to each work on their UI with minimal support and then all integrate under a single portal.

**Comment by mjashanks** [comment]
I can't see us doing this one soon unless a lot more people weigh in here!

Would it work if you put the budibase apps in an IFrame?
